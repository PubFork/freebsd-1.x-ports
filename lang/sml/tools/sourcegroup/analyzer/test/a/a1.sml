functor F (A:A) = struct
  val c = C.z
  open A
  val b = B.x
  val d = D.y
end

(* {rsi A dst A:A rst C
    (ost A [rst B rst D])}
   dfu F
*)
