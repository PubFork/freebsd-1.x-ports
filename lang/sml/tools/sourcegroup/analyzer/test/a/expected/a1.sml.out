source sml "a/a1.sml"
  import structure B C D 
  import signature A 
  export functor F ;

